brave/brave-core
# Added BR flag image to vpn panel

fix https://github.com/brave/brave-browser/issues/29719

<img width="406" alt="Screenshot 2023-04-14 at 10 15 59 PM" src="https://user-images.githubusercontent.com/6786187/232061319-41e72cf3-722f-491b-be62-1e59bd07ab3e.png">


Filed f/u issue (https://github.com/brave/brave-browser/issues/29727) - serve flag icon from leo.


Resolves 



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch with clean profile
2. Set purchased state
3. Open vpn panel and check brazil flag image
